### ISSUE TITLE
🐛 CSS parser errors on multiple semicolons

### ISSUE THREAD

**Issue by ematipico** [biomejs/biome#6485]
### Environment information

```block
`main`
```

### What happened?

The following code is valid, but Biome fails in the second semicolon:

```css
div {
  box-sizing: border-box;;
  color: red;
}
```

### Expected result

Biome shouldn't emit an error in multiple semicolons after a declaration.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
